webpack/assets/javascripts/react_app/components/common/emptyState/index.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
```

Comment:
Could you change this file so it will contain single component per file?
e.g.
* `common/EmptyState/EmptyState.js`
* `common/EmptyState/EmptyStatePattern.js`
* `common/EmptyState/EmptyStatePropTypes.js`
* `common/EmptyState/EmptyStateSecondaryActionButtons.js`
* `common/EmptyState/EmptyStatePrimaryActionButton.js`

And you can namely expose the relevant component from the `common/EmptyState/index.js`

Reply:
No problem - you got it!